scripts/tosdr.js

```diff
@@ -18,7 +18,7 @@ function getSites() {
 
             // recurse through sites list. Get and process the detailed points data for each
             getSitePoints(sites).then(result => {
-                fs.writeFile('shared/data/tosdr.js', 'const tosdr = ' + JSON.stringify(processed, null, 4), err => { if(err) console.log(err)} )
+                fs.writeFile('shared/data/tosdr.json', JSON.stringify(processed, null, 4), err => { if(err) console.log(err)} )
```

Comment:
@jdorweiler I built the release version of the extension and noticed that this script still generated the file with `const tosdr =`.

I decided to generate the file as pure JSON instead, as it's more portable. (I'm leaving the other data files as JS, since they have comments which JSON doesn't support.)